### PR TITLE
chore: unify color

### PR DESCRIPTION
**Pull request by fkakatie**
update color (background and text, including links) to align with s2 design

Test URLs:

Before: https://main--aem-boilerplate--adobe.hlx.live/
After: https://unify-color--aem-boilerplate--adobe.hlx.live/

### CODE REVIEW

**Review by rofe — COMMENTED** [review]
PSI check no longer shows 100 for accessibility due to lack of contrast...
https://pagespeed.web.dev/analysis/https-unify-color--aem-boilerplate--adobe-hlx-live/mspnh18ogw?form_factor=mobile

**Comment by rofe** [comment]
Is S2 compliance really a requirement for the boilerplate?

**Comment by fkakatie** [comment]
> Is S2 compliance really a requirement for the boilerplate?

Not at all. I'm only pulling values from it in an attempt to set some sort of visual consistency across boilerplate > block collection.

**Comment by rofe** [comment]
I think having 3 x 100 in PSI out of the box should be considered a higher priority than visual consistency.

**Comment by fkakatie** [comment]
Screenshot 2024-06-06 at 10 44 27 AM

What am I missing here? Discrepancy between `aem-code-sync` report and PSI report: https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Funify-color--aem-boilerplate--adobe.hlx.live%2F

File: styles/styles.css [hunk 1]
```diff
@@ -16,8 +16,10 @@
   --light-color: #e1e1e1;
   --dark-color: #c6c6c6;
   --text-color: #292929;
-  --link-color: #4c75ff;
-  --link-hover-color: #3b63fb;
+  --link-color: #4565F2;
+  --link-hover-color: #2f4de1;
+
+  /* #3562ff */
```
Comment (rofe) [inline styles/styles.css]: Delete?

**Comment by rofe** [comment]
> What am I missing here? Discrepancy between `aem-code-sync` report and PSI report:

I clicked re-run PSI check, now it's showing the same.

**Pull request merged**